### PR TITLE
Add block interval interface docs

### PR DESCRIPTION
**Pull request by boundless-forest**
Selection_027

#370

### CODE REVIEW

File: docs/cita/sys-contract-interface/interface.md [hunk 1]
```diff
@@ -1801,6 +1801,26 @@ $ scm SysConfig setWebsite \
         --admin-private 0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6 \
 ```
 
+### setBlockInterval
+
+动态设置出块时间。
+
+* 参数
+
+    `Integer` - 出块时间
```
Comment (kaikai1024) [inline docs/cita/sys-contract-interface/interface.md]: Show the info of time unit: `ms`?

Comment (boundless-forest) [reply docs/cita/sys-contract-interface/interface.md]: Done

**Review by kaikai1024 — APPROVED** [review]

File: docs/cita/sys-contract-interface/interface.md [hunk 2]
```diff
@@ -1801,6 +1801,26 @@ $ scm SysConfig setWebsite \
...
+    `Integer` - 出块时间(毫秒)
+
+* 返回值
+
+    空
+
+* 示例
+
+```shell
+$ scm SysConfig setBlockInterval \
```
Comment (leeyr338) [inline docs/cita/sys-contract-interface/interface.md:1819]: For a runnable shell command, It should be:

```shell
cita-cli scm ...
```
Yeah, as you magic, we need to change some commands in this file.

Comment (boundless-forest) [reply docs/cita/sys-contract-interface/interface.md:1819]: Yes！ That’s a big problem. Another pr for this flaw is nessary.
 #372

Comment (leeyr338) [reply docs/cita/sys-contract-interface/interface.md:1819]: Okay.

**Review by leeyr338 — APPROVED** [review]

**Pull request merged**